apps/src/templates/instructions/teacherFeedback/ReadonlyTeacherFeedback.jsx

```diff
@@ -0,0 +1,113 @@
+import React, {Component} from 'react';
+import i18n from '@cdo/locale';
+import PropTypes from 'prop-types';
+import color from '@cdo/apps/util/color';
+import Comment from '@cdo/apps/templates/instructions/teacherFeedback/Comment';
+import Rubric from '@cdo/apps/templates/instructions/teacherFeedback/Rubric';
+import {
+  teacherFeedbackShape,
+  rubricShape
+} from '@cdo/apps/templates/instructions/teacherFeedback/types';
+import {ReviewStates} from '@cdo/apps/templates/feedback/types';
+import ReadOnlyReviewState from '@cdo/apps/templates/instructions/teacherFeedback/ReadOnlyReviewState';
+import moment from 'moment/moment';
+
+export class ReadonlyTeacherFeedback extends Component {
+  static propTypes = {
+    rubric: rubricShape,
+    visible: PropTypes.bool.isRequired,
+    latestFeedback: teacherFeedbackShape
+  };
+
+  getLatestReviewState() {
+    const {latestFeedback} = this.props;
+    const reviewState = latestFeedback?.is_awaiting_teacher_review
+      ? ReviewStates.awaitingReview
+      : latestFeedback?.review_state;
+    return reviewState || null;
+  }
+
+  renderLastUpdated() {
+    const {created_at} = this.props.latestFeedback;
+    const formattedTime = moment.min(moment(), moment(created_at)).fromNow();
+    return (
+      <div style={styles.timeStudent} id="ui-test-feedback-time">
+        {i18n.lastUpdated()}
+        {formattedTime && (
+          <span style={styles.timestamp}>{` ${formattedTime}`}</span>
+        )}
+      </div>
+    );
+  }
+
+  render() {
+    const {rubric, visible, latestFeedback} = this.props;
+
+    if (!visible) {
+      return null;
+    }
+
+    return (
+      <div>
+        {rubric && (
+          <Rubric
+            rubric={rubric}
+            performance={latestFeedback?.performance || null}
+            isEditable={false}
+            onRubricChange={() => {}}
```

Comment:
Is `onRubricChange` required? Wondering if we can avoid passing an empty function.

Reply:
Oops, good catch, I put that in as a placeholder but it should be removed.